src/config/config.ts

```diff
@@ -91,7 +91,7 @@ enum Environment {
 }
 
 // Environment variables with defaults
-const isDev = process.env.NODE_ENV === Environment.Dev
+const isDev = process.env.NODE_ENV !== Environment.Prod
```

Comment:
this makes production less safe though - i think we should default to production if the configuration is misspecified.

Reply:
Hmm, this is because we need to differentiate between the dev and testing environment. Maybe I'll make isDev both testing and dev then.